### ISSUE TITLE
sticky nav bar - all sections display broken on iOS

### ISSUE THREAD

**Issue by michaelwmcnamara** [guardian/frontend#10220]
looking on iPhone6 - when I open all sections and try to scroll down past sport, the nav bar collapses, so I cannot see the rest of the sections.

**Comment by paperboyo** [comment]
Hmm. Why only iOS? Seems to be broken for me now on latest beta of Chrome on Android (both from main nav and from sticky situation). I remember playing with it on Windows Phone 8.1 where it was broken as well. Isn’t it broken from the very beginning of sticky (my expectations are so low, that I even don’t care to check other Android browsers I have :saddestpanda:)...

**Comment by davidrapson** [comment]
Just to second this, I've been having this problem on both iOS (Safari/iPhone 6 on iOS8) and Android (Chrome/Nexus 5 on L).

iOS does exactly what Michael describes for me but Android is unfortunately even worse where the section navigation disappears completely after scrolling for a bit and the page jumps all over the place. Both very jarring though.

**Comment by davidrapson** [comment]
Anything planned on this? Sections navigation doesn't work on small screens at all at currently. @Calanthe @uplne

**Comment by sndrs** [comment]
this has been the case for over a week. commercial's priority has been fixing display issues with ads and they haven't been able to find someone to fix it. until it's officially handed over and someone else can fix it, we're going to disable it below tablet. 

worth noting it's also broken _above_ mobile if your viewport is shorter than the expanded nav...

**Comment by OliverJAsh** [comment]
Duplicate of https://github.com/guardian/frontend/issues/10148?

**Comment by ScottPainterGNM** [comment]
Having discussed with @stephanfowler - as we will reimplement mobile before it coming back on, this bug is closed for now but needs to be considered for when we revisit this.
